gapis/api/vulkan/api/structs.api

```diff
@@ -39,23 +39,29 @@
 // Structs defined in  Vulkan 1.0 Core spec
 
 class VkApplicationInfo {
+  @values(VK_STRUCTURE_TYPE_APPLICATION_INFO)
   VkStructureType sType              /// Type of structure. Should be VK_STRUCTURE_TYPE_APPLICATION_INFO
   const void*     pNext              /// Next structure in chain
+  @len("null-terminated")
   const char*     pApplicationName
   u32             applicationVersion
+  @len("null-terminated")
   const char*     pEngineName
   u32             engineVersion
   u32             apiVersion
 }
 
 class VkInstanceCreateInfo {
+  @values(VK_STRUCTURE_TYPE_INSTANCE_CREATE_INFO)
   VkStructureType          sType                   /// Should be VK_STRUCTURE_TYPE_INSTANCE_CREATE_INFO
   const void*              pNext                   /// Pointer to next structure
   VkInstanceCreateFlags    flags
   const VkApplicationInfo* pApplicationInfo
   u32                      enabledLayerCount
+  @len("enabledLayerCount","null-terminated")
```

Comment:
This is hard to understand at a glance.
Maybe use 2 anotations
@len("enabledLayerCount") @str

Reply:
Splitted it in 2 annotations. @str and @len